include/hello_log_ids.hrl

```diff
@@ -0,0 +1,90 @@
+-define(LOGID99, {ffffffffffffffffffffffffffffffff, 999999}). %placeholder
+
+%% since erlang atoms are quoted if they begin with a number
+%% we just take UUIDs which begin with a letter. Hence we have
+%% no quoting in journald (like other message IDs).
```

Comment:
@GalaxyGorilla I see lager_journald_backend works not only with atoms https://github.com/surik/lager_journald_backend/blob/master/src/lager_journald_backend.erl#L73

What you think about if we will fix lager_journal_backend and ejournald for correct working with binaries?